src/Form/LocalistEntityForm.php

```diff
@@ -240,13 +240,13 @@ public function form(array $form, FormStateInterface $form_state) {
       '#required' => FALSE,
     ];
 
-    $form['localist_image_field_name'] = [
+    $form['localist_media_field_name'] = [
```

Comment:
Woah, it took me a WHILE to figure out that we have this field in here twice haha, I was like, "he's replacing it, but also it's still there, what is going onnnnnnnnn" -- glad that this work will have the side benefit of removing the duplicated form field declaration! 🎉